app/src/main/java/com/keylesspalace/tusky/BaseActivity.java

```diff
@@ -93,6 +97,19 @@ protected void onCreate(@Nullable Bundle savedInstanceState) {
         requesters = new HashMap<>();
     }
 
+    @Override
+    protected void attachBaseContext(Context newBase) {
+        Log.d(TAG, "attachBaseContext()");
```

Comment:
I'd prefer to not have these tracing logs in the code

Reply:
Done.